test/server/cards/02_SHD/events/Pillage.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { Location } from '../../../../../server/game/core/Constants';
```

Comment:
Unfortunately this seems to be causing test issues for me sporadically - it causes jasmine failures for some reason, though not in all cases and only if I'm running one specific file and not the whole suite. Jasmine is a pain to debug so I haven't been able to find a cause. Until we get it resolved, would you mind defaulting to just using the string values?